platform/feature-api/communication/postoffice/README.md

```diff
@@ -5,13 +5,13 @@ RPC library for TypeScript
 ## Overview
 
 This library allows transparent function calls on objects that may reside in a
-different process.  This works by capturing these calls, translating them into a
```

Comment:
These changes also seem pretty unrelated...

Reply:
yeah, this was done by the linter, I can revert it if its an issue

Reply:
Yes please, we need to fix our prettier configuration for markdown I guess! Double spaces is super weird.